### ISSUE TITLE
[Accessibility]A11y_AzureToolsStorageExplorer_Settings_DataExplorers : AI4D: (StorageExplorer/heading '') The Name property of a focusable element must not be null.

### ISSUE THREAD

**Issue by Radha2019** [microsoft/AzureStorageExplorer#1881]
“**Check out Accessibility Insights!** - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS4.1.2; #AI4D; #Win10;

**Environment Details:**
Windows Version: Windows10

Steps to Reproduce:
1. Launch Storage Explorer
2. Navigate to Setting menu on left pane and click on it.
3. Navigate to Data Explorers on right pane and  click on it.
4. "Data Explorers" screen gets opened.
5. Verify that all the elements of "Data Explorers" Screen are accessible and MAS Compliant.

**Actual:**
(StorageExplorer/heading '') The Name property of a focusable element must not be null.

**Expected:**
(StorageExplorer/heading '') The Name property of a focusable element must not be null it should be provided.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx 

**User Impact:** Screenreader user unable to know the name of the check box.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730} 

**Attachment for reference**
![1881_ Accessibility A11y_AzureToolsStorageExplorer_Settings_DataExplorers  AI4D](https://user-images.githubusercontent.com/55691739/65420839-c4494780-de1f-11e9-92a8-8e11e9d67f09.png)

**Comment by MRayermannMSFT** [comment]
Jason merged a fix for this into master

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Tools Storage Explorer (Version: 1.12.0)
The issue is no longer to reproduces, Hence closing this bug.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
